src/linter.js

```diff
@@ -448,7 +448,7 @@ export default class Linter {
     }
   }
 
-  _markBannedLibs(addonMetadata) {
+  _markBannedLibs(addonMetadata, unadvised_libraries = UNADVISED_LIBRARIES) {
```

Comment:
We generally have a pattern of a leading underscore to denote something used for DI. E.g: https://github.com/mozilla/addons-linter/blob/master/src/linter.js#L340